src/controllers/coursesController.js

```diff
@@ -1,32 +1,42 @@
 const createError = require('http-errors');
 const expressify = require('expressify')();
-const logger = require('../utils/logger.js');
+// const logger = require('../utils/logger.js');
 const coursesService = require('../services/coursesService');
 
 /**
  * Get courses.
  */
 const getCourses = async (req, res) => {
-  const { page, limit } = req.query;
+  let { page, limit } = req.query;
+  const { authorization } = req.headers;
 
-  const courses = await coursesService.getCourses({ page, limit });
+  // TODO: default values
+  page = !page ? 0 : page;
```

Comment:
you should define it in a config file. I would copy this way of define a config file: 

https://github.com/math-learning/exercises-service/tree/chores/zube-62-exercise-service/configs